### PR TITLE
Docs: Add README file

### PR DESCRIPTION
**Pull request by AStilmock**
Add README file

### CODE REVIEW

**Comment by WZale** [comment]
Looks great, Andy! Thanks for taking care of the readme and including our input.

**Comment by Jredish11** [comment]
looks swell

File: README.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+## Approach to code design
+
+  - As a group we assessed the Futbol setup pages for iterations 1 and 2
+  - All team members cloned the project repo forked by Jeff in order to review the CSV files for game/team/game-teams together. 
+  - To review the code - we compared the setup pages and the CSV files from the repo to create an outline on the Miro board to decide what classes and methods we wanted to create for the project. 
+
+
+## Link to DTR document
+
+  -https://docs.google.com/document/d/1E83-CrSe4Pgk7h_P4YHnFegHKaku1JHFG2Du9mzagmk/edit
```
Comment (Jredish11) [inline README.md:33]: hell eyah

**Review by Jredish11 — APPROVED** [review]

**Comment by jgthomas-12** [comment]
Great job Andy! Everything looks magnificent.

**Review by WZale — APPROVED** [review]
Nice!

**Pull request merged**